apps/change-log.md

```diff
@@ -14,12 +14,14 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
-## 2019-07-18
+## 2019-07-26
 
 ### Changes with software release 7.37.0
```

Comment:
@depoulo It is available in that version is it? Or will it only be available in the 7.38.0?

Reply:
Well it's in epages-ui, [versuon 2.5.0](https://github.com/ePages-de/epages-ui/releases/tag/v2.5.0).